enes-homework-week3-moreJS.js

```diff
@@ -0,0 +1,200 @@
+//Q:More JS
+
+//Q1
+
+function sum(a,b,c) {
+  return (a+b+c);
+};
+
+console.log(sum(1,2,3));
+
+//Q2
+
+function colorCar(str) {
+  console.log("a " + str + " car" );
```

Comment:
Aahaan. Can you think of an alternative for double quotes here? Perhaps using template strings(backticks) 😉

Reply:
wow thank you for reminding that. It is really better to use backticks. console.log(`a ${str} car`);